.ci/build-kit/install_and_test.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -e
+
+cmake \
+    -B build \
+    -S "$EXT_MOUNT/source" \
+    -G Ninja \
+    -DBUILD_TESTING=ON \
+    -DCMAKE_BUILD_TYPE=Debug \
+    -DCMAKE_INSTALL_PREFIX="$WORKSPACE_PATH/dist"
+
+ninja -j$(nproc) -C build install
+
+trap "cp build/Testing/Temporary/LastTest.log /ext/ctest-report" EXIT
+
+ninja -j$(nproc) -C build test
```

Comment:
The `/ext/` is specific within the docker-container. When you split-up this script in two files (`build_install.sh` and `test.sh`), you could run the `trap` inside workflow. That's not needed when running locally. (However, I hope the files here will soon be replaced with the reusable workflow anyway.)